examples/experiments/progress_bar.py

```diff
@@ -46,8 +47,10 @@
             pb.draw()
             ec.flip()
             # subject uses any key press to proceed
-            if not building_doc:
-                ec.wait_one_press()
+            ec.wait_one_press(max_wait=max_wait)
     ec.screen_text('This example is complete.')
+    screenshot = ec.screenshot()
```

Comment:
It would be a lot cooler in the gallery if you took the screenshot while the visualization was up (e.g., after line 48) on, say, the second trial

Reply:
Sorry yes, I (just) realized that the screenshot is for the docs.